Topics/Teamwork.md

```diff
@@ -11,6 +11,14 @@ Working as a team effectively requires a lot of planning and organization. Proje
 ## Team Communication
 During the early stages of forming a team, the group should also establish general expectations for how the team plans to communicate with one another. This may include setting up a team communication channel on a specific platform (ex. discord or slack) and establishing regular check-up meetings (including when these meetings are and where they will be held - for example, in person or online). These general meetings can be used to outline general expectations and project requirements, update one another on individual progress, delegate new reponsibilities, and set project deadlines. 
 
```

Comment:
A sub-header here would help signify the topic. I'd suggest using `###` going by the above header